### PR TITLE
feat: radius, aaa authentication lists, start NAT model

### PR DESCRIPTION
**Pull request by stmosher**

### CODE REVIEW

File: mdd/python/translation/openconfig_xe/xe_interfaces.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def xe_process_interfaces(self) -> None:
                             physical_interface.switchport.delete()
                         xe_interface_aggregation(self, interface, port_channel)
         # Physical and Sub-interfaces
-        elif interface.config.type == 'ianaift:ethernetCsmacd' and interface.subinterfaces.subinterface:
```
Comment (stmosher) [inline mdd/python/translation/openconfig_xe/xe_interfaces.py:141]: This is the default for ethernetCsmacd and should be kept under more specific ethernet matches.

File: test/tests/xe_system.yml [hunk 2]
```diff
@@ -8,6 +8,37 @@
   vars:
     rollback_file: "{{ lookup('env', 'PWD') }}/rollback.yaml"
 
+- hosts: localhost
+  gather_facts: no
+  tags:
+    - disable_ned_checks
+  vars:
+    device_name: xe1
+    content: |
```
Comment (stmosher) [inline test/tests/xe_system.yml:17]: Stop NED from failing on warning when command is executed anyway.

**Review by jasonking3 — DISMISSED** [review]
LGTM

**Review by jasonking3 — APPROVED** [review]
LGTM

**Pull request merged**